policyengine_us/variables/gov/states/ca/tax/income/credits/ca_cdcc.py

```diff
@@ -17,6 +17,6 @@ def formula(tax_unit, period, parameters):
             period
         ).gov.states.ca.tax.income.credits.child_dependent_care
         agi = tax_unit("adjusted_gross_income", period)
-        federal_cdcc = tax_unit("cdcc", period)
+        federal_cdcc = tax_unit("ca_federal_cdcc", period)
```

Comment:
one more thing, this decoupling only happens in 2021, from 2022 it should go back to interacting with the `cdcc`

Reply:
we can create a variable for "ca_cdcc_input" and make it a list that has the value of `ca_federal_cdcc` in 2021 and `cdcc` in 2022 and on

Reply:
can we resolve this?